src/custom_load_manager.cc

```diff
@@ -175,4 +183,11 @@ CustomLoadManager::ReadTimeIntervalsFile(
   return cb::Error::Success;
 }
 
+void
+CustomLoadManager::WaitForWarmupAndCleanup()
+{
+  RequestRateManager::WaitForWarmupAndCleanup();
+  custom_intervals_.clear();
```

Comment:
CustomLoadManager has an additional data structure that needs to be reset after warmup.